src/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wbpack Exercise</title>
+</head>
+<body>
+    <section class="to-do">
+      <ul class="to-do-list">
+        <li class="list-head">
+            <h3>Today's To Do</h3>
+            <i class="fa-solid fa-arrows-rotate"></i>
+        </li>
+        <li>
+          <input type="text" id="add-list" placeholder="Add to your list">
+          <i class="fa-solid fa-right-to-bracket add-btn" id="add-btn"></i>        
+        </li>
+        <li class="clear-completed"><span class="clear-all-completed">Clear all completed</span></li>
+      </ul>
+      <span class="clear-all-list" >Clear all Completed</span>
```

Comment:
Please make sure to add one **Clear all checked** as it is messing up your app.
![Screenshot 2022-07-19 at 11-37-03 Wbpack Exercise](https://user-images.githubusercontent.com/102610715/179709868-3bb1780e-2bd3-4741-9bc1-ce6cff6a849f.png)

Reply:
Ok so i will remove the clear all completed on the left

Reply:
yes